src/runtime/Makefile

```diff
@@ -36,6 +36,7 @@ ARCH_FILE_SUFFIX = -options.mk
 ARCH_FILE = $(ARCH_DIR)/$(ARCH)$(ARCH_FILE_SUFFIX)
 ARCH_FILES = $(wildcard arch/*$(ARCH_FILE_SUFFIX))
 ALL_ARCHES = $(patsubst $(ARCH_DIR)/%$(ARCH_FILE_SUFFIX),%,$(ARCH_FILES))
+BUILDTIME = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
This is "docker-like", but it means the final binary isn't [reproducible](https://reproducible-builds.org/), so I'd question if we need this.

Reply:
+1, maybe a commit id and if it is dirty is more appropriate.

Reply:
Ok, I have deleted `Built` item.